### ISSUE TITLE
How can I implement a 'list' to handle upcoming strings (eg. log info pipeline)?

### ISSUE THREAD

**Issue by soloradish** [gizak/termui#51]

**Comment by gizak** [comment]
Can you be more specific? (like rolling log?)

**Comment by synthesizerpatel** [comment]
@gizak While I'm not the original requestor, my interest in the feature as you describe it is what brought me here.

I noticed that you were able to get the 'list' to animate randomly in theme.go which got me thinking about using something like https://github.com/hpcloud/tail to tail a 'live' log file and then present it in a box that would only show the last nth lines, updating the window when new lines are read.

Python's urwid has a similar widget called a ListWalker (example https://github.com/wardi/urwid/blob/master/examples/edit.py#L38)

It seems like a nontrivial task to implement all the elements that would make a log window (and beyond my current go capabilities, I'm new) - so I'm not expecting anything by posting this. Just wanted to say that if that's not what the OP is requesting, I think it should be. ;)

**Comment by ns-cweber** [comment]
@soloradish @synthesizerpatel Late to the party, but I believe I just implemented this yesterday. Check out [my project][http://bitbucket.org/weberc2/hackernews] and let me know if that widget (the primary one in the application) is what you had in mind.

**Comment by mikepea** [comment]
@ns-cweber you might also want to look at my project - https://github.com/mikepea/go-jira-ui - which also has a less-like UI implemented via the List handler. Would be good to chat about where we're at and finding some common patterns :)

**Comment by ns-cweber** [comment]
@mikepea That looks very much like what I'd like to work toward (specifically with a cursor) though I got a couple of panics on resize (specifically resizing to be very small and going from full-screen to floating window in OS X). I'll take a look at your implementation.

**Comment by mikepea** [comment]
Yeah, that seems to happen when a window is resized quickly / drastically - i've not gotten to the bottom of it yet. My email address is Mike.Pountney@gmail.com if you want to chat out-of-band.
